[2020-07-01 16:42:03,317] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.get_covid_data_upload_hdfs_task 2020-07-01T11:11:39.792500+00:00 [queued]>
[2020-07-01 16:42:03,323] {taskinstance.py:669} INFO - Dependencies all met for <TaskInstance: covid_data_dag.get_covid_data_upload_hdfs_task 2020-07-01T11:11:39.792500+00:00 [queued]>
[2020-07-01 16:42:03,323] {taskinstance.py:879} INFO - 
--------------------------------------------------------------------------------
[2020-07-01 16:42:03,323] {taskinstance.py:880} INFO - Starting attempt 1 of 2
[2020-07-01 16:42:03,323] {taskinstance.py:881} INFO - 
--------------------------------------------------------------------------------
[2020-07-01 16:42:03,331] {taskinstance.py:900} INFO - Executing <Task(PythonOperator): get_covid_data_upload_hdfs_task> on 2020-07-01T11:11:39.792500+00:00
[2020-07-01 16:42:03,333] {standard_task_runner.py:53} INFO - Started process 53286 to run task
[2020-07-01 16:42:03,376] {logging_mixin.py:112} INFO - Running %s on host %s <TaskInstance: covid_data_dag.get_covid_data_upload_hdfs_task 2020-07-01T11:11:39.792500+00:00 [running]> localhost.localdomain
[2020-07-01 16:42:03,589] {logging_mixin.py:112} INFO - API Data is valid
[2020-07-01 16:42:08,863] {logging_mixin.py:112} INFO - Covid Statewise Data CSV is uploaded to HDFS
[2020-07-01 16:42:08,864] {python_operator.py:114} INFO - Done. Returned value was: None
[2020-07-01 16:42:08,868] {taskinstance.py:1052} INFO - Marking task as SUCCESS.dag_id=covid_data_dag, task_id=get_covid_data_upload_hdfs_task, execution_date=20200701T111139, start_date=20200701T111203, end_date=20200701T111208
[2020-07-01 16:42:13,312] {logging_mixin.py:112} INFO - [2020-07-01 16:42:13,311] {local_task_job.py:103} INFO - Task exited with return code 0
